src/components/PopupWithForm.js

```diff
@@ -26,7 +32,15 @@ export default class PopupWithForm extends Popup {
     });
   }
 
-  resetForm() {
-    this._formElement.reset();
+  renderLoading(isLoading, loadingText = "Saving...") {
+    if (this._submitButton) {
+      this._submitButton.textContent = isLoading
+        ? loadingText
+        : this._submitButtonText;
+    }
+  }
+
+  close() {
+    super.close();
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.